flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/HiveParser.java

```diff
@@ -306,6 +313,20 @@ private String substituteVariables(HiveConf conf, String statement) {
         return new VariableSubstitution(() -> hiveVariables).substitute(conf, statement);
     }
 
+    private Operation processAddCmd(HiveConf hiveConf, String command) {
```

Comment:
Replace the command using `substituteVariables(..)` before this method? So we don't need to pass `hiveConf` into this method.